src/ipahealthcheck/meta/core.py

```diff
@@ -2,18 +2,49 @@
 # Copyright (C) 2019 FreeIPA Contributors see COPYING for license
 #
 
+import logging
+import os
 import socket
 from ipahealthcheck.core import constants
 from ipahealthcheck.core.plugin import Result, duration
 from ipahealthcheck.meta.plugin import Plugin, registry
+from ipapython import ipautil
 from ipapython.version import VERSION, API_VERSION
 
+logger = logging.getLogger()
+
 
 @registry
 class MetaCheck(Plugin):
     @duration
     def check(self):
-        yield Result(self, constants.SUCCESS,
+
+        rval = constants.SUCCESS
+        if not os.path.exists('/usr/bin/fips-mode-setup'):
```

Comment:
Does it make sense to put that into ipaplatform.paths instead of hardcoding the path here?

Reply:
If IPA wants to add this then we could consume it pretty easily. I can open a PR against IPA but it would require an upstream release before we could utilize it.

Reply:
Can you open a PR against IPA and file an issue against freeipa-healthcheck so that we don't forget?

Reply:
freeIPA upstream issue: https://pagure.io/freeipa/issue/8429 and PR https://github.com/freeipa/freeipa/pull/4942

Reply:
https://github.com/freeipa/freeipa/pull/4942 is merged. Should we adjust things here or just file an issue against freeipa-healthcheck and do the change later (for backport reasons)?

Reply:
We need to wait until the next upstream release and then we can do it and tie the two together.